packages/block/src/constructors.ts

```diff
@@ -533,7 +533,10 @@ export async function createBlockFromBeaconPayloadJson(
  * @param cliqueSigner clique signer key to be used for creating sealed block
  * @returns the block constructed block
```

Comment:
These docs are incorrect, it returns the extraData, not a block (currently)

Reply:
Will update the docs, just copy pasted them.